src/CoreBundle/Resources/config/defaults.inc.php

```diff
@@ -335,6 +335,9 @@
 * you need to set revision management active for every table manually in table configuration also
 */
 if (!defined('CMS_ACTIVE_REVISION_MANAGEMENT')) {
+    /**
+     * @deprecated since 6.3.0
```

Comment:
```suggestion
     * @deprecated since 6.3.0 - revision management is no longer supported
```

(also for other deprecation annotations)

Reply:
done everywhere